### PR TITLE
lab6

### PR DESCRIPTION
**Pull request by TlucasX5**

### CODE REVIEW

**Comment by TlucasX5** [comment]
Week 5 session  9 says lab 6 but in grades there is a lab5 but no 6 so are these one and the same?

**Comment by TlucasX5** [comment]
I went back thru all your content on Brightspace and there is no Lab5... looks like this seq was skipped. But Lab5 is present under grades section...

File: assignments/dictionary.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+function findEmail(index, nametxt) {
+  if (nametxt in index) {
+    return index[nametxt];
+  } else {
+    return("This name is not found.");
+  }
+}
+
+function onShowEmail() {
+  const nametxt = document.getElementById('findname').value;
```
Comment (akennis) [inline assignments/dictionary.js:22]: In the future be sure to validate and normalize all user input.

**Review by akennis — APPROVED** [review]
Sorry for the confusion with the lab numbering.  Got you down for a 100% on this Lab 6.

**Pull request merged**